exporter/defoldexporter/exporter.xml

```diff
@@ -11,10 +11,10 @@
     <displayName>DefoldExporter</displayName>
 
     <!-- A description of the exporter -->
-    <description>TexturePacker to Defold exporter</description>
+    <description>Exporter for Defold</description>
 
     <!-- Exporter version -->
-    <version>1.0</version>
+    <version>1.1</version>
```

Comment:
Currently, we have the version number in two places.
SHould investigate if we can get this number in the exporter script.